cli/create.ts

```diff
@@ -64,3 +68,13 @@ export async function createPack(
     return printAndExit(errors.join('\n'));
   }
 }
+
+function parseWorkspaceIdOrUrl(workspaceIdOrUrl: string | undefined): string | undefined {
+  const urlMatch = workspaceIdOrUrl?.match(/.*\/workspaces\/ws-[a-zA-z0-9].{10}/);
```

Comment:
Hm, I don't think you need the `.` in there? That's saying 10 of any character. I think you want `[a-zA-z0-9]{10}` which is 10 of those specific characters.

Do we know that WS ids definitely don't contain dashes or underscores?

Reply:
hmmm let me check. I might have overgeneralized from my single example..

Reply:
ok yeah looks like they have dashes or underscores:

https://github.com/coda/coda/blob/main/modules/js-core/utils/ids.ts#L1